### PR TITLE
refact(upgrade): update upgrade doc and add release 3.7.0 changes to introductions

### PR DESCRIPTION
**Pull request by niladrih**
Changes:
- upgrade doc
- release update

### CODE REVIEW

**Review by pchandra19 — APPROVED** [review]
Looks ok

File: docs/main/user-guides/upgrade.md [hunk 1]
```diff
@@ -20,57 +20,139 @@ Upgrade to the latest cStor or Jiva version is supported only from 1.12.0 or lat
...
-
-2. Once the above requirements are met, execute the `helm upgrade` command to upgrade the openebs/openebs helm chart to version 3.6.0.
+Upgrade for Mayastor is supported from v2.0.x to v2.1.0 and above. Before we get into the upgrade documentation for Mayastor, it is important that we understand that there are two ways of deploying Mayastor (since v2.0.0) on to a kubernetes cluster -- the ['mayastor' helm chart](https://github.com/openebs/mayastor-extensions/tree/develop/chart) and the ['openebs' helm chart](https://github.com/openebs/charts/tree/main/charts/openebs). The upgrade instructions differ for these two deployment strategies. The first step of the upgrade is to identify the helm chart that you have used to deploy Mayastor. If you are using the Mayastor with the openebs helm chart, then the instructions for upgrading the mayastor helm chart do not apply to you.
 
-:::note
-The installation parameters present in [values.yaml](https://github.com/openebs/charts/blob/main/charts/openebs/values.yaml) can be configured based on the requirements. 
-:::
+### Indentifying the helm chart
```
Comment (avishnu) [inline docs/main/user-guides/upgrade.md:25]: typo - identifying

File: docs/main/user-guides/upgrade.md [hunk 2]
```diff
@@ -20,57 +20,139 @@ Upgrade to the latest cStor or Jiva version is supported only from 1.12.0 or lat
...
 
-3. Next, to upgrade the data plane component, select a node that has an `io-engine` pod deployed on it. To get the list of such nodes, execute
+You'll need the helm v3 binary to be on you $PATH environment variable. Execute the following command to check if you have helm v3:
```
Comment (avishnu) [inline docs/main/user-guides/upgrade.md:27]: your $PATH

File: docs/main/user-guides/upgrade.md [hunk 3]
```diff
@@ -20,57 +20,139 @@ Upgrade to the latest cStor or Jiva version is supported only from 1.12.0 or lat
...
-:::
+If the value of the 'CHART' column in the row which contains the helm chart for Mayastor, in the output of the above command, reads 'openebs-...', then your Mayastor deployment uses the openebs helm chart. If the CHART column instead reads 'mayastor-...', then it makes use of the mayastor helm chart.
+[Click here if your Mayastor deployment uses the 'mayastor' helm chart](https://mayastor.gitbook.io/introduction/additional-information/upgrade).
+
+Continue down this page if your Mayastor deployment uses the 'openebs' helm chart.
+
+### Upgrading Mayastor installed from the openebs helm chart
+
+#### 1. Planning the upgrade
+Supported upgrade paths for upgrade for Mayastor, installed with the openebs chart, are...
```
Comment (avishnu) [inline docs/main/user-guides/upgrade.md:47]: of Mayastor

File: docs/main/user-guides/upgrade.md [hunk 4]
```diff
@@ -20,57 +20,139 @@ Upgrade to the latest cStor or Jiva version is supported only from 1.12.0 or lat
...
+
+If you want to list all of the versions available for upgrade, execute the following commands:
+```
+$ helm repo update openebs             
+Hang tight while we grab the latest from your chart repositories...
+...Successfully got an update from the "openebs" chart repository
+Update Complete. ⎈Happy Helming!⎈
+
+$ # Assuming source version of 3.5.0
+$ helm search repo openebs --version ">3.5.0,>=3.7.0" -l
```
Comment (avishnu) [inline docs/main/user-guides/upgrade.md:72]: Should it be <=3.7.0 ?

File: docs/main/user-guides/upgrade.md [hunk 5]
```diff
@@ -20,57 +20,139 @@ Upgrade to the latest cStor or Jiva version is supported only from 1.12.0 or lat
...
-Ensure that there are no in-progress rebuilds.
+helm upgrade openebs openebs/openebs -n openebs --version 3.7.0 --set mayastor.image.tag=$IMG_TAG --set release.version=3.7.0 --reuse-value \
+        --set mayastor.image.repoTags.controlPlane="" \
+        --set mayastor.image.repoTags.controlPlane="" \
+        --set mayastor.image.repoTags.controlPlane="" \
+        --set mayastor.agents.core.capacity.thin.poolCommitment="250%" \
+        --set mayastor.agents.core.capacity.thin.volumeCommitment="40%" \
+        --set mayastor.agents.core.capacity.thin.volumeCommitmentInitial="40%"
+```
+The 3.6.0 release does not require special options to upgrade to 3.7.0 or above versions.
```
Comment (avishnu) [inline docs/main/user-guides/upgrade.md:138]: Just add the command instead of the statement?

**Review by avishnu — COMMENTED** [review]
Given few minor comments

File: docs/main/user-guides/upgrade.md (hunk 3 above)
Comment (niladrih) [reply docs/main/user-guides/upgrade.md:47]: Going with "Supported upgrade paths for upgrade for Mayastor..."

File: docs/main/user-guides/upgrade.md (hunk 4 above)
Comment (niladrih) [reply docs/main/user-guides/upgrade.md:72]: This is trying to list version above the user's semver version, and also above the semver version from which upgrades are supported.

File: docs/main/user-guides/upgrade.md (hunk 5 above)
Comment (niladrih) [reply docs/main/user-guides/upgrade.md]: Done..

**Review by avishnu — APPROVED** [review]

**Pull request merged**